api/src/main/java/org/pmiops/workbench/firecloud/FireCloudConfig.java

```diff
@@ -38,6 +38,9 @@
   public static final String SERVICE_ACCOUNT_GROUPS_API = "serviceAccountGroupsApi";
   public static final String SERVICE_ACCOUNT_WORKSPACE_API = "workspaceAclsApi";
   public static final String END_USER_WORKSPACE_API = "workspacesApi";
+  public static final String SERVICE_ACCOUNT_STATIC_NOTEBOOKS_API =
```

Comment:
Please update the PR description with the general approach here. I didn't expect to see FC cloud changes, but it makes sense.